crates/matrix-sdk-crypto-nodejs/src/machine.rs

```diff
@@ -109,20 +112,15 @@ impl OlmMachine {
         &self,
     ) -> Result<
         Vec<
-            Either<
+            Either5<
```

Comment:
Oh, five is the biggest so you still need (direct) nesting? This seems like sth. that should be fixed in napi 🤔

Reply:
It is yes. I will work on a patch to support up to 26 with the help of macros. We need to update 2 files: [`either.rs`](https://github.com/napi-rs/napi-rs/blob/d6c755c4eb8b2a56e226866d9a595ad2d712ebcc/crates/napi/src/bindgen_runtime/js_values/either.rs) and [`typegen.rs`](https://github.com/napi-rs/napi-rs/blob/d6c755c4eb8b2a56e226866d9a595ad2d712ebcc/crates/backend/src/typegen.rs).

Reply:
Done, https://github.com/napi-rs/napi-rs/pull/1200.